draft-ietf-tls-tls13.md

```diff
@@ -3734,20 +3763,24 @@ In this diagram, the following formatting conventions apply:
   from the left. For instance, the Early Secret is the Secret for
   generating the client_early_traffic_secret.
 
-Note that the 0-RTT Finished message is not included in the Derive-Secret
-operation.
-
 ~~~~
                  0
                  |
                  v
    PSK ->  HKDF-Extract
                  |
                  v
-           Early Secret ---> Derive-Secret(., "client early traffic secret",
+           Early Secret
+                 |
+                 +--------> Derive-Secret(., "client early traffic secret",
                  |                         ClientHello)
                  |                         = client_early_traffic_secret
-                 v
+                 |
+                 +--------> Derive-Secret(.,
+                 |                        "external psk binder key" |
+                 |                        "resumption psk binder key",
+                 |                         "")
```

Comment:
Check the alignment on these changes, I think that it's a character off on several of these lines.